draft-ietf-tls-tls13.md

```diff
@@ -809,7 +816,7 @@ contains a "key_share" extension with the server's ephemeral
 Diffie-Hellman share which MUST be in the same group as one of the
 client's shares. If PSK key establishment is
 in use, then the ServerHello contains a "pre_shared_key"
-extension indicating which of the client's offered PSKs was selected.
+extension.
```

Comment:
you need to mention the binder on line 806 (github doesn't let me comment there)